src/assets/styles/variables.scss

```diff
@@ -20,6 +20,13 @@
   --paragraph-small: 0.8rem;
   --label: 0.64rem;
   --button: 1rem;
+  --line-height: normal;
+  --letter-spacing: 0;
+  --letter-spacing-eyebrow: 0.25rem;
+  --font-family: "Lato";
```

Comment:
```suggestion
```
This line should be removed, I added user editable font variables in [this PR](https://github.com/CloudCannon/small-business-template-eleventy/pull/8)